metadata_rules/handler.py

```diff
@@ -0,0 +1,68 @@
+# handler.py
+""" Module to launch application """
+
+import _set_path  # noqa
+import os
+import json
+from pathlib import Path
+from pipelineutilities.pipeline_config import setup_pipeline_config, load_config_ssm  # noqa: E402
+import sentry_sdk  # noqa: E402
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration  # noqa: E402
+from harvest_metadata_rules import HarvestMetadataRules
+from dependencies.sentry_sdk import capture_exception
+from distutils.dir_util import copy_tree
+
+
+if 'SENTRY_DSN' in os.environ:
+    sentry_sdk.init(dsn=os.environ['SENTRY_DSN'], integrations=[AwsLambdaIntegration()])
+
+
+def run(event, context):
+    """ run the process to retrieve and process web kiosk metadata """
+    _suplement_event(event)
+    config = setup_pipeline_config(event)
+    google_config = load_config_ssm(config['google_keys_ssm_base'])
+    config.update(google_config)
+    google_credentials = json.loads(config["museum-google-credentials"])
+    harvest_metadata_rules_class = HarvestMetadataRules(google_credentials)
+    local_folder = os.path.dirname(os.path.realpath(__file__)) + '/'
+    for site_name in event['sites']:
+        harvest_metadata_rules_class.harvest_google_spreadsheet_info(site_name)
+    _save_sites_info_to_s3(local_folder + "sites/", "s3://" + config["process-bucket"] + "/sites/")
+    try:
+        copy_tree(local_folder + "sites/", local_folder + "../process_manifest/sites/")
+    except Exception:
+        print('Unable to sync sites files to process_manifest ')
+        pass
```

Comment:
if possible narrow down exception scope(Exception is too broad) and toss the error to sentry